packages/sui-js/test/server/userAgentParserSpec.js

```diff
@@ -0,0 +1,2 @@
+import test from '../userAgentParserTest'
+test()
```

Comment:
this is the only way I found to run the tests as server tests, alternatives are welcome